contracts/src/periphery/EigenDACertVerifierV2.sol

```diff
@@ -49,23 +45,23 @@ contract EigenDACertVerifierV2 is IEigenDACertVerifierV2 {
      * @notice Constructor for the EigenDA V2 certificate verifier
      * @param _eigenDAThresholdRegistryV2 The address of the EigenDAThresholdRegistry contract
      * @param _eigenDASignatureVerifierV2 The address of the EigenDASignatureVerifier contract
-     * @param _eigenDARelayRegistryV2 The address of the EigenDARelayRegistry contract
      * @param _operatorStateRetrieverV2 The address of the OperatorStateRetriever contract
      * @param _registryCoordinatorV2 The address of the RegistryCoordinator contract
      * @param _securityThresholdsV2 The security thresholds for verification
      */
     constructor(
         IEigenDAThresholdRegistry _eigenDAThresholdRegistryV2,
         IEigenDASignatureVerifier _eigenDASignatureVerifierV2,
-        IEigenDARelayRegistry _eigenDARelayRegistryV2,
         OperatorStateRetriever _operatorStateRetrieverV2,
         IRegistryCoordinator _registryCoordinatorV2,
         SecurityThresholds memory _securityThresholdsV2,
         bytes memory _quorumNumbersRequiredV2
     ) {
+        if (_securityThresholdsV2.confirmationThreshold <= _securityThresholdsV2.adversaryThreshold) {
+            revert InvalidSecurityThresholds();
+        }
```

Comment:
can we add a comment explaining why this check is required?

Reply:
I don't actually have a comment to add, as I am not sure myself why the check is required. But it is a check that existed in the old CertVerifier, so I think this issue should be out of scope here.